native_client/deepspeech.cc

```diff
@@ -809,6 +918,13 @@ DS_AudioToInputVector(const short* aBuffer,
   }
 }
 
+void 
+DS_FreeMetadata(Metadata* m) 
+{  
+  free(m->items);
```

Comment:
I guess we'd rather `delete` with `new` being done

Reply:
But items is malloc'ed in `ModelState::decode_metadata`
```C++
metadata->items = (MetadataItem*)malloc(sizeof(MetadataItem) * metadata->num_items);
```
Or are you suggesting it should be a delete if all allocation is moved to new/delete?

Reply:
Yep